examples/childffd_warning/warning_childFFD.py

```diff
@@ -0,0 +1,166 @@
+
+from __future__ import print_function
+import os
+import unittest
+import numpy
+import copy
+from baseclasses import BaseRegTest
+from pygeo import DVGeometry, geo_utils
+
+
+class RegTestPyGeo(unittest.TestCase):
+
+    N_PROCS = 1
+
+    # def setUp(self):
+        # Store the path where this current script lives
+        # This all paths in the script are relative to this path
+        # This is needed to support testflo running directories and files as inputs
+        # self.base_path = os.path.dirname(os.path.abspath(__file__))
+
+
+    def make_cube_ffd(self, file_name, x0, y0, z0, dx, dy, dz):
+        # Write cube ffd with i along x-axis, j along y-axis, and k along z-axis
+        axes = ['k', 'j', 'i']
+        slices = numpy.array(
+            # Slice 1
+            [[[[x0, y0, z0], [x0+dx, y0, z0]],
+            [[x0, y0+dy, z0], [x0+dx, y0+dy, z0]]],
+            # Slice 2
+            [[[x0, y0, z0+dz], [x0+dx, y0, z0+dz]],
+            [[x0, y0+dy, z0+dz], [x0+dx, y0+dy, z0+dz]]]],
+            dtype='d'
+        )
+
+        N0 = [2]
+        N1 = [2]
+        N2 = [2]
+
+        geo_utils.write_wing_FFD_file(file_name, slices, N0, N1, N2, axes=axes)
+
+
+
+    def test_parent_shape_child_rot(self, train=False, refDeriv=False):
+        ffd_name = '../../tests/inputFiles/small_cube.xyz'
+        self.make_cube_ffd(ffd_name, 0.1, 0.1, 0.1, 0.8, 0.8, 0.8)
+        small = DVGeometry(ffd_name, child=True)
+        small.addRefAxis('ref', xFraction=0.5, alignIndex='j')
+        
+        
+        x0 = 0.0
+        y0 = 0.0 
+        z0 = 0.0
+        dx = 1.0
+        dy = 1.0 
+        dz = 1.0
+        
+        axes = ['k', 'j', 'i']
+        slices = numpy.array(
+            # Slice 1
+            [[[ [x0, y0, z0], [x0+dx, y0, z0],    [x0+2*dx, y0, z0] ],
+            [[x0, y0+dy, z0], [x0+dx, y0+dy, z0], [x0+2*dx, y0+dy, z0]]],
+            # Slice 2
+            [[[x0, y0, z0+dz], [x0+dx, y0, z0+dz], [x0+2*dx, y0, z0+dz]],
+            [[x0, y0+dy, z0+dz], [x0+dx, y0+dy, z0+dz], [x0+2*dx, y0+dy, z0+dz]]]],
+            dtype='d'
+        )
+
+        N0 = [2]
```

Comment:
Can you just add arguments N0, N1, and N2 to the make_cube_ffd function so that you don't have code duplication here?